File: app/src/main/java/tv/hdonlinetv/besttvchannels/movies/watchfree/NetworkUtils.java
package tv.hdonlinetv.besttvchannels.movies.watchfree;

import java.util.concurrent.TimeUnit;

import okhttp3.OkHttpClient;

public class NetworkUtils {


    private static final long TIMEOUT = 30;

    public static OkHttpClient makeOkhttp() {
        OkHttpClient.Builder httpClientBuilder = new OkHttpClient.Builder()
                .readTimeout(TIMEOUT, TimeUnit.SECONDS)
                .connectTimeout(TIMEOUT, TimeUnit.SECONDS)
                .writeTimeout(TIMEOUT, TimeUnit.SECONDS);
                //.addInterceptor(new SimpleLoggingInterceptor());
        return httpClientBuilder.build();

    }
}

File: app/src/main/java/tv/hdonlinetv/besttvchannels/movies/watchfree/NetworkUtils.kt
package tv.hdonlinetv.besttvchannels.movies.watchfree

import okhttp3.OkHttpClient
import java.util.concurrent.TimeUnit

object NetworkUtils {
    private const val TIMEOUT: Long = 30

    fun makeOkhttp(): OkHttpClient {
        val httpClientBuilder = OkHttpClient.Builder()
            .readTimeout(TIMEOUT, TimeUnit.SECONDS)
            .connectTimeout(TIMEOUT, TimeUnit.SECONDS)
            .writeTimeout(TIMEOUT, TimeUnit.SECONDS)
        //.addInterceptor(new SimpleLoggingInterceptor());
        return httpClientBuilder.build()
    }
}
